### PR TITLE
Fix flocculation plant deforming when it has a cover emitting a strong redstone signal

### PR DESCRIPTION
**Pull request by NotAPenguin0**
As it turns out, cofhcore turns any redstone-powered water into a  `BlockTickingWater`, making it fail the structure check against regular water blocks. This PR fixes it by adding a new `ofAnyWater` structure element that also tests this class.

### CODE REVIEW

**Review by Alexdoru — COMMENTED** [review]
Is cofh a hard dep on GT5u ? In the multiblock fishing pong they use a reflection check to see if the mod is present. You might need to check if the mod is loaded before doing that. And once that is done would it be possible to use your same mechanic with the fishing pond multi and another one that has the same issue. Basically all multis that use a water block have the same issue

**Comment by NotAPenguin0** [comment]
Seems to be a compileOnly dep, so it would need a modloaded check yes

**Comment by NotAPenguin0** [comment]
- Guarded the class usages against a modloaded check
- Added a check for distilled water since the chemical bath accepts it
- Fixed the chemical bath counting distilled water as two blocks for some reason
- Added checks for cofhcore water classes to chemical bath and fishing pond

File: src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/MTEIndustrialWashPlant.java [hunk 1]
```diff
@@ -146,13 +149,7 @@ public IStructureDefinition<MTEIndustrialWashPlant> getStructureDefinition() {
                         .casingIndex(getCasingTextureIndex())
                         .dot(1)
                         .buildAndChain(onElementPass(x -> ++x.mCasing, ofBlock(getCasingBlock(), getCasingMeta()))))
-                .addElement(
-                    'w',
-                    ofChain(
-                        isAir(),
-                        ofBlockAnyMeta(Blocks.water),
-                        ofBlockAnyMeta(Blocks.flowing_water),
```
Comment (serenibyss) [inline src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/MTEIndustrialWashPlant.java:154]: Flowing water here seems to no longer be checked, is this intentional?

Comment (NotAPenguin0) [reply src/main/java/gtPlusPlus/xmod/gregtech/common/tileentities/machines/multi/processing/MTEIndustrialWashPlant.java:154]: It's not, oops

**Review by serenibyss — APPROVED** [review]

**Pull request merged**